docs/source/reference/services.md

```diff
@@ -30,15 +29,16 @@ Currently, these characteristics distinguish two types of Services:
 A Service may have the following properties:
 
 - `name: str` - the name of the service
-- `admin: bool (default - false)` - whether the service should have
-  administrative privileges
-- `url: str (default - None)` - The URL where the service is/should be. If a
-  url is specified for where the Service runs its own web server,
+- `url: str (default - None)` - The URL where the service is/should be.
+  If a url is specified for where the Service runs its own web server,
```

Comment:
I don't understand the URL description so well.

> If a url is specified for where the Service runs its own web server

Reply:
Took another stab at it. Let me know what you think.